components/ui/ace-editor.vue

```diff
@@ -103,6 +176,7 @@ export default {
 
     this.editor = editor
     this.cacheValue = this.value
+    this.outline.init(this.value)
```

Comment:
Ace Editor component is used for every type of request data. Not just JSON.

This makes the editor crash if the content is not JSON.

Reply:
This seems to be fixed in the new commits. Resolving this convo.